indra/newview/llappviewer.cpp

```diff
@@ -348,8 +348,6 @@ std::string gLastVersionChannel;
 LLVector3 gWindVec(3.0, 3.0, 0.0);
 LLVector3 gRelativeWindVec(0.0, 0.0, 0.0);
 
-U32 gPacketsIn = 0;
```

Comment:
This was never actually used.  It looks like old debug cruft that someone forgot to remove.